prediction/config.py

```diff
@@ -4,12 +4,17 @@
 
     Provides the flask config options
 """
-from os import getenv
+import glob
+import os
 
 
 class Config(object):
-    PROD_SERVER = getenv('PRODUCTION', False)
+    PROD_SERVER = os.getenv('PRODUCTION', False)
     DEBUG = False
+    SEGMENT_ASSETS_DIR = os.path.abspath(os.path.join(os.getcwd(), 'src', 'algorithms', 'segment', 'assets'))
+    DICOM_PATHS_DOCKER = glob.glob(os.path.join('/images_full', 'LIDC-IDRI-*', '**', '**'))
+    DICOM_PATHS_LOCAL = glob.glob(
+        os.path.join(os.getcwd(), 'src', 'tests', 'assets', 'test_image_data', 'full', 'LIDC-IDRI-*', '**', '**'))
```

Comment:
Hm, do we want to `glob.glob` on module import? Doesn't feel right.

Reply:
Does using constant wildcards as introduced in cd7d39a make more sense?